recipes/ncrf/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+mkdir -p $PREFIX/bin
+
+make 
+
+chmod +x NCRF
+
+mv NCRF $PREFIX/bin
+
+chmod +x *.py
+cp *.py $PREFIX/bin/
+ln -s ncrf_to_bed.py $PREFIX/bin/ncrf_to_bed
```

Comment:
These symlinks won't end up in the the final tarball, since they're being created in the source directory. Did you mean `ln -s ${PREFIX}/bin/ncrf_to_bed.py ${PREFIX}/bin/ncrf_to_bed`? You could probably just rename the files at that point (I don't know if users are expecting the .py extensions or not, though if they are then don't bother creating the symlinks).

Reply:
Yes, that is what I meant (ln -s ${PREFIX}/bin/ncrf_to_bed.py ${PREFIX}/bin/ncrf_to_bed).  I will change that.  Thanks.

Rename wouldn't satisfy my intent.  The goal is to satify two types of users -- those who want to run a python script with "python path/bin/ncrf_to_bed.py" and those who don't want to have to know what language it was written in with "path/bin/ncrf_to_bed". That's why I have two names for the same thing.

Reply:
Just to follow up, the previous commit would have worked I think, since the relative target of the symlink would resolve relatively to the source of the symlink which would be correct in this case.